js/index.js

```diff
@@ -1 +1,73 @@
-// Your code goes here
+const funBus = document.querySelector('.logo-heading');
+funBus.addEventListener('mouseover', function(event){
+    event.target.style.color = 'red';
+    event.target.style.fontSize = '6rem';
+});
+
+const navItems = document.querySelectorAll('.nav-link');
+navItems.forEach(function(item) {
+    console.log(item);
+    item.addEventListener('click',(item) => { item.preventDefault()});
+    item.addEventListener('mouseenter', function() {
+        console.log(event);
+        event.target.style.color = 'white';
+        event.target.style.background = 'red';
+        event.target.style.fontSize = '2.4rem';
+        event.target.style.padding = '10px 5px';
+    });  
+});
+
```

Comment:
Maybe start leaving comments to explain your code, like you could add a comment here to explain what the event is and what it does.